### ISSUE TITLE
CSV import with multiple locale seems to have trouble with duplicated names

### ISSUE THREAD

**Issue by grks** [mysociety/alaveteli#2134]
Hi

I have an issue with the import_csv function not working well on multiple locales. The organizations are created, but they aren't listed under /body for each locale.

To reproduce:

1. set AVAILABLE_LOCALES: 'nb nn en'
2. set DEFAULT_LOCALE: 'nb'
3. Import csv beneath

Expected behavior:

All 6 organizations gets listed under /body, /en/body and /nn/body. I also expect it to handle duplicate names and generate slug names without conflicts.

Current behavior:

Everything works fine on /body and /nb/body
/nn/body only has 3, 4, 5 and 6
/en/body only has 2, 3, 5 and 6

```csv
#id,request_email,name,name.nn,name.en,tag_string,home_page
23842,test@test.no,Test,Test,Test,37,http://www.test.no/
23844,test2@test2.no,Test 2,Test 2,Test 2 EN,36,http://www.test2.no/
25998,test3@test3.no,Test 3 NB,Test 3 NN,Test 3 EN,66,http://www.test3.no/
28190,test4@test4.no,Test 4,Test 4 NN,Test 4,77,http://www.test4.no/
29999,test5@test5.no,Test 5 NB,Test 5,Test 5,99,http://www.test5.no/
31000,test6@test6.no,Test 6 NB,Test 6, Test 6 EN,11,http://www.test6.no/
```

Result of import:

```
line 2: creating new authority 'Test' (locale: nb):
	{"name":"Test","request_email":"test@test.no","home_page":"http://www.test.no/","tag_string":"
37"}
line 3: creating new authority 'Test 2' (locale: nb):
	{"name":"Test 2","request_email":"test2@test2.no","home_page":"http://www.test2.no/","tag_stri
ng":"36"}
line 3: creating new authority 'Test 2' (locale: en):
	{"name":"Test 2 EN"}
line 4: creating new authority 'Test 3 NB' (locale: nb):
	{"name":"Test 3 NB","request_email":"test3@test3.no","home_page":"http://www.test3.no/","tag_s
tring":"66"}
line 4: creating new authority 'Test 3 NB' (locale: nn):
	{"name":"Test 3 NN"}
line 4: creating new authority 'Test 3 NB' (locale: en):
	{"name":"Test 3 EN"}
line 5: creating new authority 'Test 4' (locale: nb):
	{"name":"Test 4","request_email":"test4@test4.no","home_page":"http://www.test4.no/","tag_stri
ng":"77"}
line 5: creating new authority 'Test 4' (locale: nn):
	{"name":"Test 4 NN"}
line 6: creating new authority 'Test 5 NB' (locale: nb):
	{"name":"Test 5 NB","request_email":"test5@test5.no","home_page":"http://www.test5.no/","tag_s
tring":"99"}
line 6: creating new authority 'Test 5 NB' (locale: nn):
	{"name":"Test 5"}
line 6: creating new authority 'Test 5 NB' (locale: en):
	{"name":"Test 5"}
line 7: creating new authority 'Test 6 NB' (locale: nb):
	{"name":"Test 6 NB","request_email":"test6@test6.no","home_page":"http://www.test6.no/","tag_s
tring":"11"}
line 7: creating new authority 'Test 6 NB' (locale: nn):
	{"name":"Test 6"}
line 7: creating new authority 'Test 6 NB' (locale: en):
	{"name":" Test 6 EN"}
Import was successful.
```

**Comment by crowbot** [comment]
I've confirmed that this looks like a bug and made a PR to fix it.

**Comment by gorm** [comment]
Great patch! Thank you very much! I tested it on import from admin interface and seems to work now. I also tested via DRY_RUN=0 bundle exec rake import:import_csv but had some trouble without finding out what is wrong. It seems to import, but nothing gets added to db. Also tried to add RAILS_ENV=development in the vagrant image, but then I got a db error. So maybe a good idea to test from command line as well with RAILS_ENV=development DRY_RUN=0 bundle exec rake import:import_csv
